omnibus/files/private-chef-cookbooks/private-chef/recipes/erchef_database.rb

```diff
@@ -1,66 +1,53 @@
-private_chef_pg_database "opscode_chef" do
-#  owner node['private_chef']['postgresql']['username'] # Do we really want this?
-  notifies :run, "execute[chef-server-schema]", :immediately
-end
+postgres = node['private_chef']['postgresql']
 
-# Though Sqitch runs are idempotent, we need to have them run only
-# when the database is first created (thus the chained notifications).
-# This is only for EC11; subsequent versions can run them
-# idempotently, like normal.  This is just to allow fresh EC11
-# installs to use Sqitch; upgraded installations have a Partybus
-# upgrade to run.  The end result should be that whether or not you
-# are installing or upgrading EC11, at the end of the day, you'll have
-# Sqitch metadata info in your database.
-execute "chef-server-schema" do
-  command "sqitch --db-name opscode_chef deploy --verify"
-  # OSC schema is a dependency of the EC schema, and managed by it
-  cwd "/opt/opscode/embedded/service/opscode-erchef/schema/baseline"
-  user node['private_chef']['postgresql']['username']
-  # Clear PERL5LIB to ensure sqitch only uses omnibus's perl
-  # installation
-  environment "PERL5LIB" => ""
-  returns [0,1]
-  action :nothing
-  notifies :run, "execute[enterprise-chef-server-schema]", :immediately
+private_chef_pg_user postgres['sql_user'] do
```

Comment:
same here, with the correct user owning the schema we shouldn't need this any longer.